src/libpython_test/test.clj

```diff
@@ -1,9 +1,9 @@
 (ns libpython-test.test
   (:require [libpython-clj2.python :as py]))
 
-(py/initialize!)
-
-;(require '[libpython-clj2.require :as py-req])
-;(require 'libpython-clj2.metadata)
-
-(println :OK)
+(defn -main []
```

Comment:
Was this necessary to get things working? You'll see in clj-python/libpython-clj#213 that moving requirement of `libpython-clj2.require` conflicts a bit with what I'm trying to accomplish.

Reply:
no, not necessary, just for testing purposes so that `clj -m libpython-test.test` would run and print `:OK` and then exit gracefully.  Was just to verify that libpython-clj would load correctly.